src/capital-project/capital-project.repository.ts

```diff
@@ -50,6 +56,32 @@ export class CapitalProjectRepository {
           category: sql<CapitalProjectCategory>`${capitalProject.category}`,
         })
         .from(capitalProject)
+        .leftJoin(
+          cityCouncilDistrict,
+          sql`
+            ST_Intersects(${cityCouncilDistrict.liFt}, ${capitalProject.liFtMPoly})
+            OR ST_Intersects(${cityCouncilDistrict.liFt}, ${capitalProject.liFtMPnt})`,
+        )
+        .leftJoin(
+          communityDistrict,
+          sql`
+          ST_Intersects(${communityDistrict.liFt}, ${capitalProject.liFtMPoly})
+          OR ST_Intersects(${communityDistrict.liFt}, ${capitalProject.liFtMPnt})`,
+        )
+        .where(
+          and(
+            ccd ? eq(cityCouncilDistrict.id, ccd) : undefined,
+            cd
+              ? and(
+                  eq(communityDistrict.boroughId, parseInt(cd[0]).toString()),
+                  eq(
+                    communityDistrict.id,
+                    parseInt(cd.slice(1, 3)).toString().padStart(2, "0"),
```

Comment:
Separating the borough and community portions of the community district id should be done in the service, rather than in the sql query. I also think we can simplify the parsing. 

By switching our "no community district" representation to a null value, we know that if the community district is present then it must be a string with three characters. (The Zod parsing guarantees this). With this guarantee, we can simplify the extracting to:

```typescript
const boroughId = cd.slice(0,1)
const communityDistrictId = cd.(1, 3)
```

By moving this extraction to the service, we separate the concerns so that the repository/query doesn't need to know about the parsing. It also means that if there is an error while parsing, it is not obscured by our catch all "data retrieval error"